lib/createUUID.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Wow, astonishing that something like this isn't available in yarn/npm...

Reply:
there's a couple but all of this happens client-side and wanted to bypass the whole `require` snafu and also didnt want to add another dependency where a copy/paste works just fine

Reply:
Sure.